drift_dev/test/analysis/resolver/dart/enum_columns_test.dart

```diff
@@ -55,8 +64,14 @@ void main() {
 
     expect(
       file.allErrors,
-      contains(isA<DriftAnalysisError>().having((e) => e.message, 'message',
-          allOf(contains('Not an enum'), contains('NotAnEnum')))),
+      containsAllInOrder(
+        [
+          isA<DriftAnalysisError>().having((e) => e.message, 'message',
+              allOf(contains('Not an enum'), contains('NotAnEnum'))),
+          isA<DriftAnalysisError>().having((e) => e.message, 'message',
+              allOf(contains('Not an enum'), contains('NotAnEnum'))),
+        ],
+      ),
     );
```

Comment:
Do you know a better way to verify an iterable contains 2 occurrences of a matcher?

Reply:
I would just extract the matcher into a local variable and use it twice:

```dart
final notAnEnumError = isDriftError(allOf(contains('Not an enum'), contains('NotAnEnum')));

expect(file.allErrors, [notAnEnumError, notAnEnumError);
```

Reply:
I extracted the matcher in [🎨 Better type comparison and matcher](https://github.com/simolus3/drift/pull/2168/commits/4030db4c5f6eb7795f3f21c2a65065f025bdbdc0)